### ISSUE TITLE
typescript definition

### ISSUE THREAD

**Issue by ramarro123** [CreateJS/PreloadJS#242]
Hi, do you know if there is typescript definition for preloadjs 1?

**Comment by lannymcnie** [comment]
I don't think there is. We don't currently manage or update the DefinitelyTyped definitions, but plan to take that over as we roll into ES6 for the next version.

In the mean time, the API for 1.0 is very similar to 0.6.2, and should work fine.

**Comment by ramarro123** [comment]
tnx :) the d.ts of 0.6.2 it's not very good imho, everything it's object | any[] | string, making using typescript less than optimal (i can pass any params without having any compilation issue)

anyway, it work as expected with 0.6.2 definition, tnx

**Comment by ramarro123** [comment]
just to say, loadManifest for instance shouldn't be (obj | any[] | string) but
customtype | LoadItem[] | string | string[]

and customtype = {src, callback}

or i am wrong? just to enforce compilation issue of wrong params (i see in my code that someone use "id" attr as an array (in loaditem), and everything work, but id it's stated to be a string and data should be used to pass custom values)

that's why i love typescript, with strong definition, it will avoid this kind of things

**Comment by lannymcnie** [comment]
Agree, they should be better. Looking forward to supporting this officially, as we would definitely want to be as specific as possible!
